scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -108,7 +108,7 @@
         "workspaceLocation": "[replace(parameters('workspaceRegion'),' ', '')]",
         "dcrNameFull": "[Concat('MSCI', '-', variables('workspaceLocation'), '-', variables('clusterName'))]",
         "dcrName": "[if(greater(length(variables('dcrNameFull')), 64), substring(variables('dcrNameFull'), 0, 64), variables('dcrNameFull'))]",
-        "enableHighLogScaleMode": "[if(contains(parameters('streams'),'Microsoft-ContainerLogV2-HighScale'), bool('true'), bool('false'))]",
+        "enableHighLogScaleMode": "[if(or(contains(parameters('streams'),'Microsoft-ContainerLogV2-HighScale'), contains(parameters('streams'),'Microsoft-RetinaNetworkFlowLogs')), bool('true'), bool('false'))]",
```

Comment:
This API version - https://github.com/microsoft/Docker-Provider/blob/ci_prod/scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json#L468 doesnt work for AKS clusters with 1.31 version. Did you verify that, the existing API version works?

Reply:
my cluster is version 1.31.7 I did not see issue using that version, let's verify in bugbush